### ISSUE TITLE
Cannot Reproduce Post-training Results on Video-Holmes

### ISSUE THREAD

**Issue by YiwuZhong** [TencentARC/Video-Holmes#5]
Hi authors @donahowe @geyuying , 

Thanks for sharing this great work. I tried to post-train Qwen2.5-VL / Video-R1 on the training set of Video-Holmes. However, I didn't observe improvments on the test set of Video-Holmes. Could you please elaborate more details on the [reported results](https://github.com/TencentARC/Video-Holmes?tab=readme-ov-file#-training-%F0%9D%93%83%F0%9D%91%92%F0%9D%93%8C) (e.g., 27.8 --> 40.7 accuracy)?

Best,
Yiwu
